src/docs/transactions_api.dox

```diff
@@ -151,7 +157,8 @@ than snapshot, or if the current transaction has performed any write operations.
 
 \warning
 Applications not using read timestamps for search may see different results
-after the snapshot is updated.
+after the snapshot is updated. NOTE: Is this true? It shouldn't be if the set
+of actively updated timestamps are disjoint from those being read.
```

Comment:
If they aren't disjoint though? The sentence says "may", or am I missing your point?

Reply:
The question I was trying to ask here: Should we be more explicit about saying that the expected model is that writes and reads happen in disjoint ranges when using timestamps (i.e: there is no snapshot mechanism managing visibility).
Is that covered somewhere else?

Reply:
I agree most applications would be looking at disjoint ranges, but I can imagine a trigger thread looking for rows that change, and that application would loop, alternating between checking some set of k/v pairs and resetting the snapshot. (That said, I'm struggling to imagine that thread setting a read timestamp at all.)

Anyway, I don't object to the idea and it's not covered anywhere else, I don't think. Do you have suggested wording?

How about something like:
```
Applications repeatedly reading the same rows, without specifying read timestamps for search, may see
different results after the snapshot is updated.
```